cpu/msp430_common/vendor/README.md

```diff
@@ -0,0 +1,17 @@
+# Introduction
+
+The folder "msp430-gcc-support-files" has been imported from this URL:
+
+# Source
+https://software-dl.ti.com/msp430/msp430_public_sw/mcu/msp430/MSPGCC/latest/exports/msp430-gcc-support-files-${VERSION}.zip
+Check msp430-gcc-support-files/Revisions_Header.txt for the actual version.
+
+# How to update
+
+The script `update.sh` can be used to update to the newest version.
+
+The steps would be:
+
+1. figure out newest version, update URL in `update.sh`
+2. run update.sh
+3. create a commit with the changes, PR that
```

Comment:
This would be a good candidate for @aabadie approach of using a package to obtain the vendor files. But this could be done as follow up just as well.

Reply:
not entirely the same, as this comes from a http url, which we don't nicely cache at the moment.

Reply:
Let's put this on the "worry about this after merging this PR" list. But it would be nice to have caching for HTTP URLs anyway, as other PKGs might use those as well.